lib-ce/emqx_modules/src/emqx_mod_delayed.erl

```diff
@@ -234,3 +259,13 @@ do_publish(Key = {Ts, _Id}, Now, Acc) when Ts =< Now ->
 
 -spec(delayed_count() -> non_neg_integer()).
 delayed_count() -> mnesia:table_info(?TAB, size).
+
+clean_by_clientid(ClientId) ->
+    CurrNode = node(),
+    Nodes = ekka_mnesia:running_nodes(),
+    lists:foreach(fun(Node) when Node =:= CurrNode ->
+                      do_clean_by_clientid(ClientId);
+                     (Node) ->
+                      rpc:call(Node, ?MODULE, do_clean_by_clientid, [ClientId], 5000)
+                  end,
+                  Nodes).
```

Comment:
use `multicall`.
also we should add a comment why multicall (because the table is a `local_content`